crates/ironrdp-client/Cargo.toml

```diff
@@ -34,25 +34,25 @@ ironrdp-tokio.workspace = true
 sspi = { workspace = true, features = ["network_client", "dns_resolver"] } # enable additional features
 
 # GUI
-softbuffer = "0.2.0"
-winit = "0.28.3"
+softbuffer = "0.3"
+winit = "0.28"
 
 # CLI
 clap = { version = "4.2", features = ["derive", "cargo"] }
-exitcode = "1.1.2"
-inquire = "0.6.0"
+exitcode = "1.1"
+inquire = "0.6"
```

Comment:
`cargo upgrade` (`cargo-edit`) is now able to handle versions without always adding the patch number

Reply:
🔥